api/python/quilt3/session.py

```diff
@@ -297,3 +298,20 @@ def create_botocore_session():
         botocore_session.register_component('credential_provider', resolver)
 
     return botocore_session
+
+
+def get_boto3_session() -> boto3.Session:
+    """
+    Return a Boto3 session with Quilt credentials.
+    If no Quilt credentials are found, return a session that uses normal credentials provider chain.
+
+    > Note: you need to call `quilt3.config("https://your-catalog-homepage/")` to have region set,
+    if you previously called it in quilt3 < 6.1.0.
+    """
+    # XXX: maybe it makes more sense to return None if
```

Comment:
yeah i'd do this (or maybe add an optional argument to fallback to the "system" session?)